# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/remoteworkers/RemoteWorkerDebuggerManagerChild.cpp
# Commit: 01fe98c08d10
# Full Hash: 01fe98c08d102c28e3a61209ea31834eff83202a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-25 21:02:33
# Regressor Bug: 1899503
# File Overlap Count: 4
# Description:
#   Bug 1899503 - P3 PRemoteWorkerDebugger binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebugger binding.
#   
#   We are trying to keep the same logic debugger registration/unregistration to minimize the difference between the local debugger(the debugger on the content process main thread)
# ==============================================================================

diff -r 9bd2a82481f4 -r 01fe98c08d10 dom/workers/remoteworkers/RemoteWorkerDebuggerManagerChild.cpp
--- a/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerChild.cpp	Tue Mar 25 09:13:20 2025 +0000
+++ b/dom/workers/remoteworkers/RemoteWorkerDebuggerManagerChild.cpp	Tue Mar 25 09:13:20 2025 +0000
@@ -13,10 +13,6 @@
       RemoteWorkerService::Thread()->IsOnCurrentThread());
 }
 
-RemoteWorkerDebuggerManagerChild::~RemoteWorkerDebuggerManagerChild() {
-  MOZ_ASSERT_DEBUG_OR_FUZZING(
-      RemoteWorkerService::Thread() &&
-      RemoteWorkerService::Thread()->IsOnCurrentThread());
-}
+RemoteWorkerDebuggerManagerChild::~RemoteWorkerDebuggerManagerChild() {}
 
 }  // end of namespace mozilla::dom